workspace/notebook/nsip_representation.json

```diff
@@ -145,66 +145,56 @@
 					"            'AGENT'\r\n",
 					"        WHEN RR.representationFrom = 'An Organisation' THEN \r\n",
 					"            'ORGANISATION'\r\n",
-					"        WHEN RR.representationFrom = 'Members of the Public/Businesses' and SU2.Organisation IS NOT NULL THEN \r\n",
+					"        WHEN RR.representationFrom = 'Members of the Public/Businesses' and SU2.Organisation IS NOT NULL THEN\r\n",
 					"            'ORGANISATION'\r\n",
-					"        WHEN RR.representationFrom = 'Members of the Public/Businesses' THEN \r\n",
+					"        WHEN RR.representationFrom = 'Members of the Public/Businesses' THEN\r\n",
 					"            'PERSON'\r\n",
-					"        WHEN RR.representationFrom = 'Another Individual or Organisation' and SU2.Organisation IS NOT NULL THEN \r\n",
-					"            'ORGANISATION'\r\n",
-					"        WHEN RR.representationFrom = 'Another Individual or Organisation' THEN \r\n",
+					"        WHEN RR.representationFrom = 'Another Individual or Organisation' and SU2.Organisation IS NOT NULL THEN\r\n",
+					"             'ORGANISATION'\r\n",
+					"        WHEN RR.representationFrom = 'Another Individual or Organisation' THEN\r\n",
 					"            'PERSON'\r\n",
-					"        WHEN RR.representationFrom = 'Myself' THEN \r\n",
+					"        WHEN RR.representationFrom = 'Myself' THEN\r\n",
 					"            'PERSON'\r\n",
-					"        ELSE \r\n",
-					"            RR.representationFrom\r\n",
-					"    END AS representationFrom   \r\n",
-					"    -- ,CASE \r\n",
-					"    --     WHEN SU.serviceUserType = 'RepresentationContact' THEN \r\n",
-					"    --         RR.NSIPRepresentaionID\r\n",
-					"    --     ELSE \r\n",
-					"    --         ''\r\n",
-					"    -- END AS representedId\r\n",
-					"    -- ,CASE \r\n",
-					"    --     WHEN SU2.serviceUserType = 'Agent'THEN \r\n",
-					"    --         RR.agentcontactid\r\n",
-					"    --     ELSE \r\n",
-					"    --         ''\r\n",
-					"    -- END AS representativeId\r\n",
-					"    ,COALESCE(RR.representedId, '') AS representedId\r\n",
-					"    ,COALESCE(RR.representativeId, '') AS representativeId\r\n",
+					"        ELSE\r\n",
+					"            RR.RelRepOnBehalfOf\r\n",
```

Comment:
RR.RelRepOnBehalfOf does not exist. It is renamed as RR.representationFrom to match the service bus schema